core_project_message.md

```diff
@@ -0,0 +1,41 @@
+Operate software in a production grade environment
```

Comment:
You can format titles in markdown by adding `#` , or for subheadings `##` or for sub-sub-headings `###` etc. Example: 

For example if you rephrase the title in line 1 to: 

` # Operate software in a production grade environment`

It will then render as: 

# Operate software in a production grade environment